modules/luci-base/po/zh-cn/base.po

```diff
@@ -3797,7 +3806,7 @@ msgid ""
 "or Safari."
 msgstr ""
 "您的 IE 浏览器太老了，无法正常显示这个页面！请更新到 IE7 及以上或使用其他浏览"
-"器，例如：Chrome、Firefox、Opera、Safari。"
+"器，如Firefox、Opera、Safari。"
```

Comment:
Why drop out Chrome?

The final punctuation should be a period in this case because it ends with alphabetical words.  I remembered someone told me this but I am not professional in this regard.

Reply:
cause you didn't mention Chrome in the original text.

I think it should be full-width period because you treat Firefox, Opera, Safari as proper nouns.

Reply:
Okay, just find an example in section 5.1.2.1 at [1] to suggest using the full-width width circle period, though I still thought it's better to use dot period in this case, just like we use period when the sentence ends with numericals

 [1] http://www.moe.gov.cn/ewebeditor/uploadfile/2015/01/13/20150113092346124.pdf

Reply:
This usage is removed from the latest GB/T 15834-2011.

[1] https://web.archive.org/web/20121020111229/http://www.sasac.gov.cn/n1180/n1271/n6716920/n6716950/n6723785/6723901.html , Section 4.1.1.
[2] http://www.moe.gov.cn/ewebeditor/uploadfile/2015/01/13/20150113091548267.pdf , Section 4.1.2.